dev/packages/example/nginx-1.2.0/manifest.yml

```
@@ -27,37 +27,34 @@ datasources:
         # This is for selection in the stream
         # id: nginx
         type: nginx/metrics
-        description: Collecting metrics for nginx.
+        # Title should never have an ending dot.
+        title: Collecting metrics from nginx hosts
+        description: Collectiong stubstatus metrics from the nginx hosts.
```

```suggestion
        title: Collect metrics from nginx hosts
        description: Collect stubstatus metrics from nginx hosts.
```

copy correction to match with title and description copy of `logs` input (L58-59)